### PR TITLE
Add a script to show the download count on an external LED panel

### PR DESCRIPTION
**Pull request by hughsie**
This is really a toy, and is only used to make myself feel warm and fuzzy.

IMG_20200215_134629

@superm1 if you want me to send you some hardware, let me know -- I made one spare just in case. :)

### CODE REVIEW

**Comment by superm1** [comment]
Ha, nice! Appreciate the thought, but I probably can skip the hardware this go around for it. You have a backup in case it breaks.

File: contrib/led/90-lvfs-led.rules [hunk 1]
```diff
@@ -0,0 +1 @@
+ACTION=="add", KERNEL=="ttyUSB*", TAG+="systemd", ENV{SYSTEMD_WANTS}+="lvfs-led@%k.service"
```
Comment (superm1) [inline contrib/led/90-lvfs-led.rules]: Anything more informative to tag? This would set off for example a zwave controller too

**Comment by hughsie** [comment]
> You have a backup in case it breaks

Well, my daughter is currently making it scroll her name, so.... :)

File: contrib/led/90-lvfs-led.rules (hunk 1 above)
Comment (hughsie) [reply contrib/led/90-lvfs-led.rules]: I guess I could match on idVendor=0403, idProduct=6001 -- but it's really the generic FTDI chip in the cable. Out of interest, what's the vendor ID of the other device?

Comment (superm1) [reply contrib/led/90-lvfs-led.rules]: I was mixing up another dongle, this one is ttyACM0.

**Pull request merged**